docs/getting-started/wallets.md

```diff
@@ -51,11 +59,13 @@ The Bittensor Command Line Interface (BTCLI) and Bittensor Python SDK offer more
 
 ## Coldkey details
 
-In `btcli`, the coldkey is equivalent to the wallet name. For example, the `--wallet.name` option in a `btcli` command always accepts only `<coldkey>` as its value and the `--wallet.hotkey` option only accepts `<hotkey>` as its value. This is because the coldkey holds the permissions and ownership over multiple hotkeys on-chain; hence, the wallet name is assigned to the coldkey.
+In `btcli`, the coldkey is equivalent to the wallet name. For example, the `--wallet.name` option in a `btcli` command always accepts only `<coldkey>` as its value and the `--wallet.hotkey` option only accepts `<hotkey>` as its value. 
```

Comment:
As above, not sure if the intent was to break up these multiline paragraphs/stanzas into single line stanzas, but your change didn't do that, and I don't think they should be broken up anyway (for example, "this" refers back to the previous sentence, definitely should not start a new paragraph...), so please those newlines.

Reply:
nope, not intending to break up. Just putting each sentence on its own line for tidyness and more precise reviewing. Then you can select just one sentence at a time if you want when reviewing a document.